pandas/core/strings.py

```diff
@@ -68,29 +76,29 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
     When ``na_rep`` is `None` (default behavior), NaN value(s)
```

Comment:
I would also say something here about not passing `other` that then all values in the Series are concatenated in a single string (so to indicate you first give an example of this part of the behaviour of the method)

Reply:
Added :-)

Reply:
Did you add this?